chart/templates/malcolm_configmaps.yaml

```diff
@@ -149,21 +152,21 @@ data:
   discovery.seed_hosts: {{ template "malcolm.discoverySeeds" . }}
   # OpenSearch index patterns and timestamp fields
   # Index pattern for network traffic logs written via Logstash (e.g., Zeek logs, Suricata alerts)
-  MALCOLM_NETWORK_INDEX_PATTERN: "arkime_sessions3-*"
+  MALCOLM_NETWORK_INDEX_PATTERN: "{{ .Values.siem_env.malcolm_network_index_pattern }}"
   # Default time field to use for network traffic logs in Logstash and Dashboards
-  MALCOLM_NETWORK_INDEX_TIME_FIELD: "firstPacket"
+  MALCOLM_NETWORK_INDEX_TIME_FIELD: "{{ .Values.siem_env.malcolm_network_index_time_field }}"
   # Suffix used to create index to which network traffic logs are written (supports Ruby strftime strings in %{})
-  MALCOLM_NETWORK_INDEX_SUFFIX: "%{%y%m%d}"
+  MALCOLM_NETWORK_INDEX_SUFFIX: "{{ .Values.siem_env.malcolm_network_index_suffix }}"
   # Index pattern for other logs written via Logstash (e.g., nginx, beats, fluent-bit, etc.)
-  MALCOLM_OTHER_INDEX_PATTERN: "malcolm_beats_*"
+  MALCOLM_OTHER_INDEX_PATTERN: "{{ .Values.siem_env.malcolm_other_index_pattern }}"
   # Default time field to use for other logs in Logstash and Dashboards
-  MALCOLM_OTHER_INDEX_TIME_FIELD: "@timestamp"
+  MALCOLM_OTHER_INDEX_TIME_FIELD: "{{ .Values.siem_env.malcolm_other_index_time_field }}"
   # Suffix used to create index to which other logs are written (supports Ruby strftime strings in %{})
-  MALCOLM_OTHER_INDEX_SUFFIX: "%{%y%m%d}"
+  MALCOLM_OTHER_INDEX_SUFFIX: "{{ .Values.siem_env.malcolm_other_index_suffix }}"
   # Index pattern used specifically by Arkime (will probably match MALCOLM_NETWORK_INDEX_PATTERN, should probably be arkime_sessions3-*)
-  ARKIME_NETWORK_INDEX_PATTERN: "arkime_sessions3-*"
```

Comment:
`ARKIME_NETWORK_INDEX_PATTERN` and `ARKIME_NETWORK_INDEX_TIME_FIELD` will probably never change (and are only, at the moment, used by the [malcolm api](https://idaholab.github.io/Malcolm/docs/api.html#API)) so it's not as critical that you allow them to be customized, but it's fine that they're here.

Reply:
No change leaving as default in values file.